package.json

```diff
@@ -61,6 +61,14 @@
       }
     ],
     "commands": [
+      {
+        "command": "onevscode.refresh-one-explorer",
+        "title": "Refresh",
+        "icon": {
+          "light": "media/icon/light/refresh.svg",
+          "dark": "media/icon/dark/refresh.svg"
```

Comment:
https://github.com/Samsung/ONE-vscode/blob/886c32ef0cc17aafefa5372506bceb7a9a83155f/package.json#L83-L86

what about this?

Reply:
Much better thank ;-D